src/main/java/eu/yals/controllers/YalsErrorController.java

```diff
@@ -0,0 +1,169 @@
+package eu.yals.controllers;
+
+import eu.yals.Endpoint;
+import eu.yals.constants.App;
+import eu.yals.constants.Header;
+import eu.yals.constants.MimeType;
+import eu.yals.exception.error.YalsError;
+import eu.yals.json.YalsErrorJson;
+import eu.yals.utils.AppUtils;
+import eu.yals.utils.ErrorUtils;
+import eu.yals.utils.YalsErrorKeeper;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.web.servlet.error.ErrorController;
+import org.springframework.stereotype.Controller;
+import org.springframework.transaction.CannotCreateTransactionException;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@Slf4j
+@Controller
+public class YalsErrorController implements ErrorController {
+    private final String TAG = "[Error Controller]";
+
+    private final YalsErrorKeeper errorKeeper;
+    private final ErrorUtils errorUtils;
+
+    HttpServletRequest request;
+    HttpServletResponse response;
+
+    Throwable rawException;
+    Throwable cause;
+    int status;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5392475)